static/js/components/dashboard/FinalExamCard.js

```diff
@@ -175,12 +202,15 @@ export default class FinalExamCard extends React.Component<void, Props, void> {
       navigateToProfile,
       submitPearsonSSO,
       pearson,
+      ui: { dialogVisibility: { pearsonTosDialogVisible = false } },
+      showPearsonTOSDialog
     } = this.props;
 
     if (!SETTINGS.FEATURES.EXAMS) {
       return null;
     }
 
+    let dialog = renderPearsonTOSDialog(pearsonTosDialogVisible, showPearsonTOSDialog, submitPearsonSSO);
```

Comment:
could we call this in the `schedulableCard` function instead? as is we're just going to be dropping it on the floor most of the time (I mean that we're calling the `renderPearsonTOSDialog` function but if the `pearson_exam_status` is not `PEARSON_PROFILE_SCHEDULABLE` we aren't going to render it, so it's not necessary).